src/utils/address.test.ts

```diff
@@ -0,0 +1,37 @@
+import {getAvailableNetworks} from 'context/NetworkContext';
+import {NetworkType} from 'src/types';
+import {isAddressValid, toShortAddress} from './address';
+
+const POLKADOT_ADDRESS = '1HDgY7vpDjafR5NM8dbwm1b3Rrs4zATuSCHHbe7YgpKUKFw';
+const KUSAMA_ADDRESS = 'D3KNzSrENcvScLGYKdzMNu7LBcCsoVyZ8X8DDjSh6Sg6LXN';
+const LITMUS_ADDRESS = 'jcSKTfG892DzZbXJjm13MRNptERKHEDXpsE8ooUFPt9pz8KvJ';
+
+test('toShortAddress function test', () => {
+  const testCase = [POLKADOT_ADDRESS, KUSAMA_ADDRESS, LITMUS_ADDRESS];
+  const expected = ['1HDgY7…pKUKFw', 'D3KNzS…Sg6LXN', 'jcSKTf…pz8KvJ'];
+  testCase.forEach((address, i) => {
+    const decimalValue = toShortAddress(address);
+    expect(decimalValue).toBe(expected[i]);
+  });
+});
+
+test('isAddressValid function test', () => {
+  const networkTypes = getAvailableNetworks();
+  const validAddressPolka = isAddressValid(networkTypes[0] as NetworkType, POLKADOT_ADDRESS);
+  expect(validAddressPolka).toBeTruthy();
```

Comment:
```suggestion
  expect(validAddressPolka).toBe(true);
```
In all these cases you should assert that it is `true`/`false`.. If we break the function like make it to return a `string` this test will not catch that